### PR TITLE
feat(pci-kubernetes): add flavor selector

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/pci-kubernetes
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | DTCORE-2447
| License          | BSD 3-Clause

## Description

Add flavor selection component

### CODE REVIEW

**Review by sidlynx — APPROVED** [review]

File: packages/manager/apps/pci-kubernetes/src/constants.ts [hunk 1]
```diff
@@ -2,3 +2,58 @@ export const NAME_INPUT_CONSTRAINTS = {
...
 };
+
+// @TODO move to pci-common
+export const LOCAL_ZONE_INFO_URL = {
+  DEFAULT: 'https://ovhcloud.com/en/public-cloud/local-zone-compute/',
+  ASIA: 'https://ovhcloud.com/asia/public-cloud/local-zone-compute/',
+  DE: 'https://ovhcloud.com/de/public-cloud/local-zone-compute/',
+  ES: 'https://www.ovhcloud.com/es-es/public-cloud/local-zone-compute/',
+  IE: 'https://ovhcloud.com/en-ie/public-cloud/local-zone-compute/',
+  IT: 'https://ovhcloud.com/it/public-cloud/local-zone-compute/',
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager/apps/pci-kubernetes/src/constants.ts:30]: done ty

File: packages/manager/apps/pci-kubernetes/src/constants.ts [hunk 2]
```diff
@@ -2,3 +2,58 @@ export const NAME_INPUT_CONSTRAINTS = {
...
+};
+
+// @TODO move to pci-common
+export const GLOBAL_REGIONS_INFO_URL = {
+  DEFAULT: 'https://ovhcloud.com/en/public-cloud/compute/',
+  ASIA: 'https://ovhcloud.com/asia/public-cloud/compute/',
+  DE: 'https://ovhcloud.com/de/public-cloud/compute/',
+  ES: 'https://www.ovhcloud.com/es-es/public-cloud/compute/',
+  IE: 'https://ovhcloud.com/en-ie/public-cloud/compute/',
+  IT: 'https://ovhcloud.com/it/public-cloud/compute/',
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager/apps/pci-kubernetes/src/constants.ts:56]: done ty

File: packages/manager/apps/pci-kubernetes/public/translations/flavors/Messages_de_DE.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager/apps/pci-kubernetes/public/translations/flavors/Messages_de_DE.json:1]: yes those files are copied directly from the angularjs apps ; there is no translation request associated

**Comment by frenautvh** [comment]
Add a commit to move flavor-selector component to manager-pci-common

**Review by qpavy — APPROVED** [review]

**Review by kqesar — APPROVED** [review]

**Pull request merged**